torch/csrc/distributed/rpc/Message.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include <torch/csrc/distributed/rpc/rpc_headers.h>
+#include <torch/serialize.h>
+#include <vector>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+enum MessageType {
+  BUILTIN_OP = 0,
+  BUILTIN_RET,
+  PYTHON_UDF_OP,
+  PYTHON_UDF_RET,
+  SHUTDOWN,
+  UNKNOWN
```

Comment:
Why do we even have an unknown message type??

Reply:
We use it when `FutureMessage` is not expected to hold any real message, e.g., when sending a no-response request or sending a response. This type can be dropped after making `FutureMessage::message_` optional.